README.md

```diff
@@ -1,2 +1,29 @@
-# targomo-jackson-datatype-trove
-A module to serialize trove collection with the popular JSON libray
+# jackson-datatype-trove
+This module facilitates serialization and deserialization of certain Trove objects. So far it supports:
+* TIntIntMap
+* TIntObjectMap
+* TObjectIntMap
+
+This is a forked project from https://bitbucket.org/marshallpierce/jackson-datatype-trove.
+It has been slightly amended to work with our jackson version and we added a few serializers/deserializers. 
+In the future we will potentially add more of them.
+
+To include add to maven: 
+```
+<dependency>
+    <groupId>com.targomo</groupId>
+    <artifactId>jackson-datatype-trove</artifactId>
+    <version>0.0.2</version>
+</dependency>
+```
+To include to your Jackson Object Mapper execute:
+```
+ObjectMapper om = new ObjectMapper();
+om.registerModule(new JodaModule())
```

Comment:
das joda verwirrt hier

Reply:
oder bruachen wir das? steht ja auch in der pom.xml? ist das nicht nur für dates/times?